### DISCUSSION TITLE
Mandate for Timeliness of date for INN and ONN files

### DISCUSSION THREAD

**Discussion by ABPhase2** [CMSgov/price-transparency-guide#562] [Q&A]
All-

Please assist with the timeliness mandate of the data on the INN and ONN files. Is it acceptable for the data on the file to be as of previous month while posting for the current month ? 

Examples specific to vendor data - 
1) Vendor data till July 31st will be published on the website for September 1st, vendors will run data extracts during the month of Aug ( with data till July 31st) to be published on Sept 1st. Is this an acceptable scenario ? 

Any additional insights will be helpful to gain clarity around this subject.

**Comment by shaselton-usds** [comment]
Question 37 on the [TiC technical clarification website](https://www.cms.gov/healthplan-price-transparency/resources/technical-clarification) tries to address some of the timing questions, hopefully including the one you asked here.
